.github/workflows/tav-command.yml

```diff
@@ -72,15 +72,38 @@ jobs:
               }
               if (match[1]) {
                 if (match[1] !== 'all') {
-                  modules = match[1].split(',')
+                  modules = match[1].split(',').map((name) => modules.find((m) => m.name === name))
                 }
               }
               if (match[3]) {
-                versions = match[3].split(',')
+                versions = match[3].split(',').map(Number)
               }
             }
-            core.setOutput('modules', modules)
-            core.setOutput('versions', versions)
+
+            const badModuleIndex = modules.findIndex((m) => !m)
+            const badVersionIndex = versions.findIndex((v) => isNaN(v))
+
+            if (badModuleIndex !== -1) {
+              core.setFailed(`Incorrect modules list, please review it'`)
+              return
+            }
+            if (badVersionIndex !== -1) {
+              core.setFailed(`Incorrect versions list, please review it'`)
+              return
+            }
```

Comment:
nit: I think the processing here could be a little more helpful and include the unknown module or version in the error message. Also it could perhaps handle *multiple* unknown modules and versions.  `/test tav foo,bar,baz`